src/callbacks.c

```diff
@@ -399,7 +403,7 @@ gboolean on_motion (GtkWidget *win,
           if (type == GROMIT_LINE)
             {
 	      draw_line (data, ev->device, devdata->lastx, devdata->lasty, ev->x, ev->y);
-              if (devdata->cur_context->arrowsize > 0)
+              if (devdata->cur_context->arrowsize > 0.0)
```

Comment:
does it matter?

Reply:
I think this makes it clearer that arrowsize is a float, not an int. The compiled code will of course be the same.

Reply:
OK :innocent: